src/Layout/toolset-tasks/toolset-tasks.csproj

```diff
@@ -2,13 +2,14 @@
   <PropertyGroup>
     <TargetFrameworks>$(SdkTargetFramework);net472</TargetFrameworks>
     <TargetFrameworks Condition=" '$([MSBuild]::IsOSPlatform(`Windows`))' == 'false' ">$(SdkTargetFramework)</TargetFrameworks>
+    <CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>
```

Comment:
😕 Why did this one need to change?

Reply:
The new version of the nuget wasn't being copied so we were getting build failures